patterns/bottlerocket/eks.tf

```diff
@@ -0,0 +1,143 @@
+################################################################################
+# EKS Cluster
+################################################################################
+# This DataSource is for testing purposes in order to validate Botterocket Update Operator
+data "aws_ami" "eks_bottlerocket" {
+  most_recent = true
+  owners      = ["amazon"]
+
+  filter {
+    name   = "name"
+    values = ["bottlerocket-aws-k8s-1.28-x86_64-v1.15*"]
+  }
+}
+
+module "eks" {
+  source  = "terraform-aws-modules/eks/aws"
+  version = "~> 19.21"
+
+  cluster_name                   = local.name
+  cluster_version                = "1.28"
+  cluster_endpoint_public_access = true
+
+  cluster_enabled_log_types = ["api", "audit", "authenticator", "controllerManager", "scheduler"]
+
+  vpc_id     = module.vpc.vpc_id
+  subnet_ids = module.vpc.private_subnets
+
+  cluster_addons = {
+    coredns = {
+      most_recent = true
+    }
+    kube-proxy = {
+      most_recent = true
+    }
+    vpc-cni = {
+      most_recent    = true
+      before_compute = true
+    }
+  }
+
+  manage_aws_auth_configmap = true
+  aws_auth_roles = [
+    {
+      rolearn  = module.eks_blueprints_addons.karpenter.node_iam_role_arn
+      username = "system:node:{{EC2PrivateDNSName}}"
+      groups = [
+        "system:bootstrappers",
+        "system:nodes",
+      ]
+    },
+  ]
+
+  eks_managed_node_group_defaults = {
+    ami_type       = "BOTTLEROCKET_x86_64"
+    instance_types = ["m5.large", "m5a.large"]
+
+    iam_role_attach_cni_policy = true
+  }
+
+  eks_managed_node_groups = {
+    bottlerocket = {
+      platform = "bottlerocket"
+
+      # The following specifies a custom ami_id, this can be provided by a data-source. Remove/comment to use the latest Bottlerocket AMI available.
+      ami_id = data.aws_ami.eks_bottlerocket.image_id
```

Comment:
instead of this, can we just use `ami_release_version` and specify an older release version to start? that would keep everything "native" within managed nodegroups realm, and you won't need to inject the bootstrap user data, etc.